cli/src/program.rs

```diff
@@ -169,7 +169,7 @@ pub enum ProgramCliCommand {
         use_lamports_unit: bool,
         bypass_warning: bool,
     },
-    ExtendProgram {
+    ExtendProgramChecked {
         program_pubkey: Pubkey,
         additional_bytes: u32,
     },
```

Comment:
We'll need the `authority_signer_index` added to this, since this only works if the payer happens to be the upgrade authority